### PR TITLE
Minor SQL Fixes

### LINKED ISSUE #415: [QUEST][WOTLK] Killing Two Scourge With
**Description**: Burning Skeleton drops fire on any target

**Current behaviour**: Currently on the quest **Killing Two Scourge With** will drop fire on any target/mob that you attack. This means if you attack any mob on the way towards your **Chained Abomination** you will need to get a new **Burning Skeleton**.

**Expected behaviour**: Should only be applied on the **Chained Abomination(s)**.

**Steps to reproduce the problem**:

1. .quest add 13144
2. Click on a **Grasping Arm**.
3. Start fighting a mob, around the area.

IMG1

**Branch(es)**: 0.x / 1.x / master (Specify the branch(es) affected by this issue) 

**AC hash/commit**:  e04409ee23d2

**Operating system**:  Windows 7

**Modules**:  No

**Other Customizations**: No

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing c++ create a Pull Request)

### PR DESCRIPTION
**Pull request by starrheld**
**Changes proposed:**

-  Fixes **[NPC]Lord Thorval's** speech
- **[NPC]Nedar, Lord of Rhinos** should no longer evade when you kill "Lunchbox"
- Some love for the **Oracles** in **Rainspeaker Canopy**
- Emotes for **[NPC][WOTLK] Pitch**
- Pathing for **[NPC][WOTLK] Venomtip**
- Fixed quest **[QUEST][WOTLK] Killing Two Scourge With**
- **[NPC][TBC] Ferfalen Druid** will now transform in to an Owl instead of a Bear.

**Target branch(es):** master

**Issues addressed:** Closes #415 

**Tests performed:** Tested, works! Need some feedback pls :1st_place_medal: 

**NOTE** You no longer need to squash your commits, on merge we will squash it for you. (GitHub added a new feature)

### CODE REVIEW

**Comment by falzard** [comment]
I tested the PR. I suppose he should say something but he just walks and go in front of his disciples but says nothing.

**Comment by falzard** [comment]
Tested and it works fine. No more error in console and he says his speech.

**Comment by Rushor** [comment]
source for this is https://github.com/dalaranwow/dalaran-wow so give credits

**Comment by starrheld** [comment]
@Rushor I did, but tried to find an email for that one, but couldn't find. Give me Teridan-dal-wow's email and I'll glady add it.

**Comment by Rushor** [comment]
use plain e-amail: Teridan-dal-wow <Teridan-dal-wow@users.noreply.github.com>

**Comment by starrheld** [comment]
@Rushor 
This is what I have
IMG

**Comment by falzard** [comment]
You know, last time I tested it because you said ".go creature xxxx and speak to him" but this time I have no idea how to test it. You could provide more infos

**Comment by Rushor** [comment]
yes ok, good enough, the author is correct now.

File: data/sql/updates/pending_db_world/rev_1488880633534999000.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1488880633534999000');
```
Comment (Rushor) [inline data/sql/updates/pending_db_world/rev_1488880633534999000.sql]: that's not your fix, give credits if you copy paste stuff from https://github.com/dalaranwow/dalaran-wow (https://github.com/dalaranwow/dalaran-wow/issues/4523)

File: data/sql/updates/pending_db_world/rev_1488923537207948700.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1488923537207948700');
+-- Burning Skeleton SAI
+SET @ENTRY := 31048;
```
Comment (Rushor) [inline data/sql/updates/pending_db_world/rev_1488923537207948700.sql]: not your fix, give credits

File: data/sql/updates/pending_db_world/rev_1488880461400153400.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1488880461400153400');
```
Comment (Rushor) [inline data/sql/updates/pending_db_world/rev_1488880461400153400.sql]: again not you fix, give credits to the author from https://github.com/dalaranwow/dalaran-wow/issues/4523

**Comment by HeartWell** [comment]
@Rushor Will you always point to the author? Forget about it, most people do not need it. Yes, and how would a person not be able to make a similar fix? The more so the decision can be enjoyed.

**Comment by Rushor** [comment]
> Will you always point to the author?

yep

**Comment by HeartWell** [comment]
@Rushor Do this not who will not.

**Comment by Rushor** [comment]
what? i hate it if someone simply steals the code of someone else without adding a thanks text from the orginal author or adding the correct authorname in the commit.

It's like you steal sth. in the market and if the police demands you to stop doing it you will say: "Forget about it, most people do not need it."

It's all about respect and fairness.

**Comment by HeartWell** [comment]
And we humans can not think the same way? But you do not believe most of the fixes of the database are similar for everyone, and now we will look for who wrote what? Or do you have a psychic and you know who wrote it?

**Comment by Rushor** [comment]
what?  look at this https://github.com/azerothcore/azerothcore-wotlk/pull/412/commits/b257ad5be144be2ac290f11d6807d3a6d4958240 and look at this https://github.com/dalaranwow/dalaran-wow/issues/4523#issue-211785849

It's exactly the same code and starrheld even copied the same whitespace error in the description of "Pathing for Venomtip". It's **not** similar, it's the **same code** because he simply copypasted it without giving credits - and that's simply rude.

**Comment by falzard** [comment]
Thank you @Rushor for giving out publicly dalaran-wow fixes, much appreciated, even if it's only for 30 minutes!

**Pull request closed without merging**